source/vibe/http/common.d

```diff
@@ -601,7 +601,7 @@ struct CookieValueMap {
 
 	inout(string)* opBinaryRight(string op)(string name) inout if(op == "in")
 	{
-		foreach(c; m_entries)
+		foreach(ref c; m_entries)
```

Comment:
Travis seems to suggest everything is good all the way down to libevent 2.065 so this seems like the proper way to go